File: py/IEC61968/Common/PersonRole.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Core.IdentifiedObject import IdentifiedObject


class PersonRole(IdentifiedObject):

    def __init__(self):
        super().__init__()
        self.appointments = None
        self.configuration_events = None

File: java/IEC61968/Common/PersonRole.java
package IEC61968.Common;

import IEC61970.Base.Core.IdentifiedObject;

/**
 * @author T. Kostic
 * @version 1.0
 * @created 25-Dec-2023 8:45:23 PM
 */
public class PersonRole extends IdentifiedObject {

	/**
	 * All appointments for this person.
	 */
	public Appointment Appointments;
	/**
	 * All configuration events created for this person role.
	 */
	public ConfigurationEvent ConfigurationEvents;

	public PersonRole(){

	}
}//end PersonRole